Some obscure test fails...

____________________________ test_get_word_lengths _____________________________
    def test_get_word_lengths():
        text = "Three tomatoes are walking down the street"
>       assert get_word_lengths(text) == [5, 8, 3, 7, 4, 3, 6]
E       assert None == [5, 8, 3, 7, 4, 3, ...]
E        +  where None = get_word_lengths('Three tomatoes are walking down the street')
example.py:29: AssertionError

--- a/example.py
+++ b/example.py
@@ -19,7 +19,7 @@ def get_word_lengths(s):
     the word lengths in string s.
     """
     # uncomment next line in step 9
-#   return [len(word) for word in s.split()]
+    return [len(word) for word in s.split()]
     return None
 
 

